etc/config/vb.amazon.properties

```diff
@@ -2,16 +2,27 @@ compilers=&vb
 supportsBinary=false
 needsMulti=false
 compilerType=vb
-defaultCompiler=dotnet601vb
+defaultCompiler=dotnet700vb
 
-group.vb.compilers=dotnet601vb
+group.vb.compilers=dotnet601vb:dotnet6011vb:dotnet700vb
 
-compiler.dotnet601vb.exe=/opt/compiler-explorer/dotnet-v6.0.1/bin/coreclr/Linux.x64.Checked/.dotnet/dotnet
+compiler.dotnet601vb.exe=/opt/compiler-explorer/dotnet-v6.0.1/tests/coreclr/Linux.x64.Checked/Tests/Core_Root/.dotnet/dotnet
```

Comment:
No, please fix the installer. I don't want to have `tests` in the path. We can fix this now and I can kick off another build rather than hackily refer to "tests" in this configuration

Reply:
Or if you can't fix the builder/installer please let me know and I'll take a look at fixing it.

Reply:
We just need the Core_Root directory. So I think we can only make the tarball against the Core_Root directory in build-dotnet.sh.

Reply:
PR sent: https://github.com/compiler-explorer/misc-builder/pull/53

The remaining thing is to update the path in *.properties to reflect the tarball structure change.